.github/workflows/config/spellcheck_config.yml

```diff
@@ -157,11 +157,14 @@ matrix:
       # Ignore content between inline backticks
       - open: '(?P<open>`+)'
         close: '(?P=open)'
-      # Ignore the first lowercase word after @param, which is usually the parameter name
-      - open: '@param(?:\s[a-z]+)?'
+      # Ignore the first word after \command or @command, which is a code artifact
+      - open: '[\\@]([abcep]|em)(?:\s\S+)?'
```

Comment:
I believe `\a`, `\e`, and `em` are for italics. Why don't we want to spellcheck the word in italics? (I don't believe they will always be code artifacts.)

I believe `\b` is for bold. Why don't we want to spellcheck the word in bold? (Similarly, I don't believe they will always be code artifacts.)

Reply:
The problem is that they can be mathematical expressions and not in CamelCase. Simply put, the spellchecking is too restrictive.

Reply:
> The problem is that they can be mathematical expressions and not in CamelCase. Simply put, the spellchecking is too restrictive.

Are you saying that `\a`, `\e`, `\em`, `\b` are being used for mathematical expressions? That would be a bit surprising to me, but maybe I'm missing something.

FWIW, it might be worth taking a look at #1060, as that new PR makes all of current files with similar (but slightly different) modifications than are in this PR. It seems like eliminating all checks for things that we put in bold are italics might be too permissive and lead to missed spelling errors.

Reply:
`\a` is short for argument, so I think it better allow any legal C++ identifier word after it. Spell checking that is overly conservative.

I would use `\e` to write simple references to variables in formula if I could. Since it doesn't work, I don't write that documentation.

I think the small pinhole that this makes in the spell checking is slightly better than enforcing a Draconian spellcheck that doesn't let me write doxygen documentation in a natural way.